frontend/src/utils/moment.js

```diff
@@ -0,0 +1,622 @@
+//
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+'use strict'
+
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+import duration from 'dayjs/plugin/duration'
+import customParseFormat from 'dayjs/plugin/customParseFormat'
+import utc from 'dayjs/plugin/utc'
+import timezone from 'dayjs/plugin/timezone'
+
+dayjs.extend(relativeTime)
+dayjs.extend(duration)
+dayjs.extend(customParseFormat)
+dayjs.extend(utc)
+dayjs.extend(timezone)
+dayjs.extend(timezoneNames)
+
+function timezoneNames (option, dayjsClass, dayjsFactory) {
+  const tzNames = [
```

Comment:
This is list what `moment.tz.names()` returns. Dayjs does not have this method yet because the Intl API does not support this yet (see https://github.com/tc39/proposal-intl-enumeration). I think there is no other option than to bundle list list in our code. There is no reason to update this list very often.